index.js

```diff
@@ -71,6 +71,9 @@ Barrels.prototype.associate = function(collections, done) {
         Model.findOne(that.idMap[modelName][itemIndex]).exec(function(err, model) {
           if (err)
             return nextItem(err);
+          
+          if(!model)
```

Comment:
Space missing.

Reply:
what space?

Reply:
Space after `if`. It's really not a big deal, but let's stay consistent.